### DISCUSSION TITLE
转出的视频，只能potplayer播放器播放，无法用其它播放器

### DISCUSSION THREAD

**Discussion by PandaZuo** [ywmoyue/biliuwp-lite#394] [bug]
### Bug 描述

转出的视频，只能potplayer播放器播放，无法用其它播放器

### 操作步骤

```plain text
转出的视频，只能potplayer播放器播放，无法用其它播放器
```

### 期望结果

转出的视频，只能potplayer播放器播放，无法用其它播放器

### 应用来源

Github/其他Github镜像站

### 代码贡献

我更擅长发现问题

**Comment by ywmoyue** [comment]
下载的视频是h264还是h265的

**Comment by PandaZuo** [comment]
第3个，av什么来着，导出后是mp4格式的



---原始邮件---
发件人: ***@***.***&gt;
发送时间: 2023年11月7日(周二) 下午5:32
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [ywmoyue/biliuwp-lite] 转出的视频，只能potplayer播放器播放，无法用其它播放器 (Discussion #394)




 
下载的视频是h264还是h265的

**Reply by ywmoyue** [reply]
那需要在应用的设置-播放中点击 安装AV1扩展

**Reply by ywmoyue** [reply]
安装完扩展之后就可以用Windows自带播放器播放
如果说使用其他播放器播放不了，那可能是其他播放器不支持AV1编码

**Comment by PandaZuo** [comment]
好的，我试试，同时也也试试其它格式



---原始邮件---
发件人: ***@***.***&gt;
发送时间: 2023年11月7日(周二) 晚上6:06
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [ywmoyue/biliuwp-lite] 转出的视频，只能potplayer播放器播放，无法用其它播放器 (Discussion #394)




 
安装完扩展之后就可以用Windows自带播放器播放
 如果说使用其他播放器播放不了，那可能是其他播放器不支持AV1编码
